examples/java-agent/src/main/java/example/HelloWorld.java

```diff
@@ -17,9 +17,16 @@
 package example;
 
 import static spark.Spark.get;
+import static spark.Spark.port;
 
 public class HelloWorld {
   public static void main(String[] args) {
-    get("/hello", (req, res) -> "Hello World");
+    // Allow use with Cloud Run which requires listening on the value in PORT
+    String portEnv = System.getenv("PORT");
```

Comment:
Is it defined as 4567 by default somewhere? It should continue to work locally.

Reply:
SparkJava listens on 4567 by default (mentioned in the README.md)